src/charts/helpers/filter.js

```diff
@@ -133,7 +133,24 @@ define(function (require) {
         return filterId;
     };
 
+    const createCircleHighlight = (el, ease, radius, bounceRadius = radius * 2) => {
```

Comment:
Nit: maybe we could call it `bounceCircleHighlight`, as it is not really creating it, right?